librz/include/rz_basefind.h

```diff
@@ -17,11 +17,33 @@ extern "C" {
 #define RZ_BASEFIND_SCORE_MIN_VALUE   (1)
 
 typedef struct rz_basefind_t {
-	ut64 candidate;
-	ut32 score;
+	ut64 candidate; ///< Candidate physical base address
+	ut32 score; ///< Score of the candidate address
 } RzBaseFindScore;
 
-RZ_API RZ_OWN RzList *rz_basefind(RZ_NONNULL RzCore *core, ut32 pointer_size);
+typedef struct rz_basefind_info_t {
+	ut32 thread_idx; ///< Thread number
+	ut32 n_threads; ///< Total number of threads
+	ut64 begin_address; ///< Thread begin address
+	ut64 current_address; ///< Thread current address
+	ut64 end_address; ///< Thread end address
```

Comment:
What is the "thread begin address'?

Reply:
each thread has a range to scan, that is the thread range

Reply:
ok maybe be a bit more explicit about this in the doc for these fields.

Moreover, from a more high level point of view, why do we care about the details? I mean, do we really need to export the information about threads running the search? Are you planning to export this info in cutter?

Reply:
yes.